en/components/grid-excel-style-filter.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Grid Excel Style Filter - Grid Feature
+_description: The Grid Excel Style Filter provides an overlay for column manipulations such as sorting, filtering, column moving, hiding and pinning.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
```

Comment:
Shall we add Excel Style Filtering as a keyword?

Reply:
I agree with you but keywords are defined by marketing for SEO and we are not allowed to touch them :)